ci/prow/Makefile

```diff
@@ -41,6 +41,8 @@ update-boskos-config: get-cluster-credentials
 update-cluster: get-cluster-credentials
 	kubectl apply -f cluster.yaml
 
+update-full: get-cluster-credentials update-cluster update-config update-boskos
```

Comment:
should we consider add a target for update config-start.yaml?

Reply:
Ah..  I didn't notice that before. LimitRange stuff should go in cluster.yaml, not in cluster_start.yaml.

Reply:
there is still a need for applying config_start.yaml, like this one: https://github.com/knative/test-infra/pull/888

Reply:
This is a rare case. I prefer not having a rule to apply config_start.yaml, so whoever has to do it really knows that it's needed (instead of someone blindly applying the rule).

/hold cancel